core/src/main/java/org/apache/calcite/rel/rel2sql/SqlImplementor.java

```diff
@@ -1057,9 +1057,9 @@ public Builder builder(RelNode rel, Clause... clauses) {
           needNew = true;
         }
       }
-      if (rel instanceof LogicalAggregate
+      if (rel instanceof Aggregate
```

Comment:
@pavelgubin are there other implications of widening this? What about enumerable aggregates, would they appear here?

Reply:
It was wrong from in the first place because `JdbcAggregate` is not `LogicalAggregate`. I presume that only `JdbcAggregate`s can appear here but the logic itself pretty valid for any `Aggregate`. So I'm widened in defensive manner.